Tests/Tracking/APITest.swift

```diff
@@ -65,6 +65,19 @@ class TrackingAPITest: UnitTest {
         CustomerIO.shared.screen(name: "", data: encodableData)
         mock.screen(name: "", data: encodableData)
 
+        // register push token
+        CustomerIO.shared.registerDeviceToken("")
+        mock.registerDeviceToken("")
+
+        // delete push token
+        CustomerIO.shared.deleteDeviceToken()
+        mock.deleteDeviceToken()
+
+        // track push metric
+        let metric = Metric.delivered
+        CustomerIO.shared.trackMetric(deliveryID: "", event: metric, deviceToken: "")
+        mock.trackMetric(deliveryID: "", event: metric, deviceToken: "")
```

Comment:
New public functions for customers to use if they wish.

Reply:
noticed and made a comment in another file regarding if there is a need for this pr to introduce these?